.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Managed
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  managed-build:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+    
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+    
+    - name: Setup MSBuild.exe
+      uses: microsoft/setup-msbuild@v1.0.2
+    
+    - name: Restore
+      run: nuget restore src\BehaviorsSdk.sln
+    
+    - name: Build
+      run: msbuild src\BehaviorsSdk.sln /p:Configuration=Release /p:SignType=real
+    
+    - name: Run Tests
+      run: dotnet test Test\UnitTests\UnitTests.csproj
+
+    - name: Pack Behaviors
+      if: success()
+      run: msbuild src\Microsoft.Xaml.Behaviors\Microsoft.Xaml.Behaviors.csproj /p:Configuration=Release -t:pack
+    
+    - name: Upload NuGet Artifacts
```

Comment:
Should we remove the Upload step now that we have this in the Azure Pipeline?

Reply:
I was wondering about that too

Reply:
Never mind. This isn't uploading anything to nuget.org but just keeps these build artifacts for GH workflow runs, so we should keep it.

Reply:
From GitHub Copilot :-)

> GitHub stores artifacts in the "Actions" tab of your repository. Here's how you can view them:
> 
> Navigate to the Repository: Go to the repository where the workflow runs.
> 
> Access the Actions Tab: Click on the "Actions" tab at the top of the repository page.
> 
> Select a Workflow Run: Find and click on the workflow run that generated the artifacts. You can identify it by the workflow name and the date it ran.
> 
> View Artifacts: Once you're on the workflow run page, scroll down to the "Artifacts" section. Here you will see a list of artifacts that were uploaded during the workflow run. You can download them from this section.
> 
> You can also access them directly from the workflow run summary page.